### DISCUSSION TITLE
Support for multiple seasonal periods?

### DISCUSSION THREAD

**Discussion by drmario-gh** [pycaret/pycaret#1895] [Q&A]
The `seasonal_period` argument of `setup` only allows to specify a single value. However, several estimators such as TBATS or prophet support multiple seasonalities directly, and I guess the rest could support them via multiple deseasonalizing. I couldn't find anything about this on the roadmap so I was wondering: is there a way to work with multiple seasonal periods in pycaret that I missed?

**Comment by moezali1** [comment]
@ngupta23

**Comment by ngupta23** [comment]
Let's track it here: https://github.com/pycaret/pycaret/issues/1901
